src/gatsby/hooks/useOnEscape.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
```

Comment:
Great idea putting the escape logic into a hook! It makes the popup hook a lot simpler and we are able to reuse this logic, if needed, elsewhere!